projects/akkadotnet.md

```diff
@@ -0,0 +1,17 @@
+# Akka.NET
+
+[Akka.NET](https://getakka.net) is a set of open-source libraries for designing scalable, resilient systems that span processor cores and networks. Akka allows you to focus on meeting business needs instead of writing low-level code to provide reliable behavior, fault tolerance, and high performance.
+
+## Project Details
+* [Project Info Site](https://getakka.net/) 
+* [Project Code Site](https://github.com/akkadotnet/akka.net)
+* Project License Type: [Apache License 2.0](https://aspnetwebstack.codeplex.com/license)
```

Comment:
Might need to change this to https://github.com/akkadotnet/akka.net/blob/dev/LICENSE?

Reply:
good catch, thanks.